test/e2e/framework/pod/cgroups.go

```diff
@@ -0,0 +1,395 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pod
```

Comment:
I was speaking to @pohly about giving node approvers approval in the framework/pod directory, and we agreed that for utilities that are only used for pod and node tests, it would be better to just move them out of the framework package.

I think we should put this in `test/e2e/common/node` instead, and maybe name it something like `cgroup_helpers.go` to distinguish from tests. Alternatively, we could add a new subpackage for helpers there instead.

Reply:
BTW, I'm working on moving `resize.go` too, so don't worry about the circular dependency there.

Reply:
I ended up adding a new package, `test/e2e/common/node/helpers`. See https://github.com/kubernetes/kubernetes/pull/131267